MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+                                    TextColor="White"
+                                   FontSize="Default"
+                                   VerticalTextAlignment="Center"
+                                   HorizontalTextAlignment="Start"
+                                   LineBreakMode="NoWrap"
+                                   HorizontalOptions="FillAndExpand">
+                                            <Label.Style>
+                                                <Style TargetType="Label">
+                                                    <Setter Property="FontAttributes" Value="Italic" />
+                                                </Style>
+                                            </Label.Style>
+                                        </Label>
+                                    </StackLayout>
+                                </DataTemplate>
+                        </sfgrid:DataGridSummaryColumn.Template>
+                    </sfgrid:DataGridSummaryColumn>
+                </sfgrid:DataGridSummaryRow.SummaryColumns>
+            </sfgrid:DataGridSummaryRow>
+        </sfgrid:SfDataGrid.CaptionSummaryRow> 
+    </sfgrid:SfDataGrid>
+
+{% endhighlight%}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+
+// To write a converter, follow the code example:
+
+
+public class GroupCaptionConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var data = value != null ? value as Group : null;
+        if (data != null)
+        {
+            SfDataGrid dataGrid = (SfDataGrid)parameter;
+            var summaryText = SummaryCreator.GetSummaryDisplayTextForRow((value as Group).SummaryDetails, dataGrid.View);
+
+            return summaryText;
+        }
+        return null;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return null;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with template loaded for individual caption summary column](Images\Caption-Summary\TemplateColumn.png)
+
+## Group summary
+
+Group summary values are calculated based on records in the group. The summary information will be displayed at the bottom of each group. You can view the group summary row by expanding the corresponding group header. The data grid adds any number of group summary row.
+
+Add group summary rows in the data grid by adding the [DataGridSummaryRow]() to [SfDataGrid.GroupSummaryRows]() collection.
+
+### Displaying summary in the row
+
+The summary information can be displayed in the row by setting the [DataGridSummaryRow.ShowSummaryInRow]() to `true` and by defining summary columns. You have to define the [DataGridSummaryRow.Title]() based on the [DataGridSummaryRow.Name]() property to format summary columns value in a row.
+ 
+Refer to [Formatting Summary](#formatting-summary) section to know more about how to format summary. 
```

Comment:
Summary information can be displayed in the entire row by setting the [DataGridSummaryRow.ShowSummaryInRow](https://chat.openai.com/) property to true and defining summary columns. The [DataGridSummaryRow.Title](https://chat.openai.com/) property needs to be defined based on the [DataGridSummaryColumn.Name](https://chat.openai.com/) property in order to format the values of the summary columns in a row.

Please refer to the [Formatting Summary](https://chat.openai.com/#formatting-summary) section to learn more about how to format the summary

Reply:
Modified.